### ISSUE TITLE
Add tests if autogenerated config files are valid

### ISSUE THREAD

**Issue by grzanka** [yaptide/converter#17]
This should be done in separate PR once #12 is merged.

The idea is following:

- [ ] use wget or something similar to download shieldhit demo executable into `tests/res` directory
- [ ] add pymchelper to requirements
- [ ] use pymchelper to run `shieldhit` with single particle (should be fast) on the configs produced by tests
- [ ] check if exit code means success or not

This way we ensure ourselves that the config files produced by a parser can be safely digested by shieldhit.

This probably requires further update of `pymchelper` (error catching and exit code handling)
